.github/workflows/build_docs.yml

```diff
@@ -0,0 +1,36 @@
+name: build_docs
+
+on: [push]
```

Comment:
So muesste das dann, wenn er nur beim Merge/Push in den Master laufen soll

```suggestion
on:
  push:
    branches:
      - master
```

Reply:
Ist eingebracht - Danke